tools/travis/prepare-tests.sh

```diff
@@ -49,6 +49,9 @@ if [[ "$DISABLE_PATH_INFO" == "1" ]]; then
 	sed -i -e "s/disable_path_info = Off/disable_path_info = On/" config.inc.php
 fi
 
+# Replaces the mailer by a mocked instance
+sed -i -e "s/default = sendmail/default = disabled/" config.inc.php
```

Comment:
Update the tests to disable the mail delivery, perhaps it will bring some performance improvement, but my motivation was to avoid error messages.

As I wrote in the issue, setting this to "log", will pollute the error_log with emails.